{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Process Federation Server",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "18.0.0.1"
                        },
                        {
                            "status": "affected",
                            "version": "18.0.0.2"
                        },
                        {
                            "status": "affected",
                            "version": "19.0.0.1"
                        },
                        {
                            "status": "affected",
                            "version": "19.0.0.2"
                        },
                        {
                            "status": "affected",
                            "version": "19.0.0.3"
                        }
                    ]
                },
                {
                    "product": "Automation Workstream Services in Cloud Pak for Automation",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "19.0.0.3"
                        }
                    ]
                }
            ],
            "datePublic": "2020-04-01T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "The IBM Process Federation Server 18.0.0.1, 18.0.0.2, 19.0.0.1, 19.0.0.2, and 19.0.0.3 Global Teams REST API does not properly shutdown the thread pools that it creates to retrieve Global Teams information from the federated systems. As a consequence, the Java Virtual Machine can't recover the memory used by those thread pools, which leads to an OutOfMemory exception when the Process Federation Server Global Teams REST API is used extensively. IBM X-Force ID: 177596."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 6.5,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 5.7,
                        "temporalSeverity": "MEDIUM",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/C:N/S:U/A:H/PR:L/I:N/AV:N/AC:L/UI:N/RC:C/E:U/RL:O",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Denial of Service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2020-04-02T14:20:21",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6125403"
                },
                {
                    "name": "ibm-icp4a-cve20204325-dos (177596)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/177596"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "psirt@us.ibm.com",
                    "DATE_PUBLIC": "2020-04-01T00:00:00",
                    "ID": "CVE-2020-4325",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "Process Federation Server",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "18.0.0.1"
                                                    },
                                                    {
                                                        "version_value": "18.0.0.2"
                                                    },
                                                    {
                                                        "version_value": "19.0.0.1"
                                                    },
                                                    {
                                                        "version_value": "19.0.0.2"
                                                    },
                                                    {
                                                        "version_value": "19.0.0.3"
                                                    }
                                                ]
                                            }
                                        },
                                        {
                                            "product_name": "Automation Workstream Services in Cloud Pak for Automation",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "19.0.0.3"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "IBM"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "The IBM Process Federation Server 18.0.0.1, 18.0.0.2, 19.0.0.1, 19.0.0.2, and 19.0.0.3 Global Teams REST API does not properly shutdown the thread pools that it creates to retrieve Global Teams information from the federated systems. As a consequence, the Java Virtual Machine can't recover the memory used by those thread pools, which leads to an OutOfMemory exception when the Process Federation Server Global Teams REST API is used extensively. IBM X-Force ID: 177596."
                        }
                    ]
                },
                "impact": {
                    "cvssv3": {
                        "BM": {
                            "A": "H",
                            "AC": "L",
                            "AV": "N",
                            "C": "N",
                            "I": "N",
                            "PR": "L",
                            "S": "U",
                            "UI": "N"
                        },
                        "TM": {
                            "E": "U",
                            "RC": "C",
                            "RL": "O"
                        }
                    }
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "Denial of Service"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://www.ibm.com/support/pages/node/6125403",
                            "refsource": "CONFIRM",
                            "title": "IBM Security Bulletin 6125403 (Automation Workstream Services in Cloud Pak for Automation)",
                            "url": "https://www.ibm.com/support/pages/node/6125403"
                        },
                        {
                            "name": "ibm-icp4a-cve20204325-dos (177596)",
                            "refsource": "XF",
                            "title": "X-Force Vulnerability Report",
                            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/177596"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2020-4325",
        "datePublished": "2020-04-01T00:00:00",
        "dateReserved": "2019-12-30T00:00:00",
        "dateUpdated": "2020-04-02T14:20:21",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}